lib/bolt/transport/sudoable/file_commands.rb

```diff
@@ -0,0 +1,59 @@
+# frozen_string_literal: true
+
+module Bolt
+  module Transport
+    class Sudoable < Base
+      class FileCommands
```

Comment:
I think this object is easier to understand if we call it "a Tmpdir" rather then "a FileCommands"